Azure/sonic-buildimage
# [docker-teamd]: Manage teamd and teamsyncd processes with supervisor

**- What I did**
 - Ensure all teamd and teamsyncd processes inside docker-teamd are managed by supervisor to ensure we receive syslog messages if any of these processes exit unexpectedly.

**- How I did it**
 - Generate supervisor config file dynamically from a Jinja2 template using docker-teamd-init.sh to accommodate variances in LAG configuration.

**- How to verify it**
 - Start / stop teamd docker (via `systemctl start/stop teamd`) and ensure all processes start/stop properly, and that teamd processes clean up after themselves properly.
- Kill teamd process(es) via signals and ensure supervisor writes appropriate syslog messages.

**- A picture of a cute animal (not mandatory but encouraged)**

```
¸.·´¯`·.´¯`·.¸¸.·´¯`·.¸><(((º>
```
